### PR TITLE
Basic CI setup

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Basic CI setup

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

Done on CI

## Checklist
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary

### CODE REVIEW

**Review by juandspy — APPROVED** [review]

**Review by juandspy — CHANGES_REQUESTED** [review]
CI is failing due to no .go files

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
   include:
     - stage: build
       script:
-        - go build
+        - Make build
```
Comment (juandspy) [inline .travis.yml:23]: `make` in lowercase?

Comment (tisnik) [reply .travis.yml]: I know, this is wip you know :)

Comment (juandspy) [reply .travis.yml]: oops, didn't see the WIP

**Review by juandspy — APPROVED** [review]

**Pull request merged**